leapp/libraries/stdlib/__init__.py

```diff
@@ -166,16 +166,20 @@ def run(args, split=False, callback_raw=_console_logging_handler, callback_lineb
     :rtype: dict
     """
     if not args:
-        raise ValueError('Command to call is missing.')
+        message = 'Command to call is missing.'
+        api.current_logger().error(message)
+        raise ValueError(message)
     api.current_logger().debug('External command has started: {0}'.format(str(args)))
     _id = str(uuid.uuid4())
     result = None
     try:
         create_audit_entry('process-start', {'id': _id, 'parameters': args, 'env': env})
         result = _call(args, callback_raw=callback_raw, callback_linebuffered=callback_linebuffered, env=env)
         if checked and result['exit_code'] != 0:
+            message = 'Command {0} failed with exit code {1}.'.format(str(args), result.get('exit_code'))
+            api.current_logger().error(message)
```

Comment:
This is not necessarily an error and should not be logged as such. Maximum debug as well, as the caller needs to handle the exception and deal with it how to treat it.

Reply:
It's already being raised as `CalledProcessError`; I'm just adding the logging part which is simply consistent with that notion.

Reply:
(Come to think about it, the error message could be useful if the CPE is _not_ caught, ie. somewhere back in leapp.  That would be PR for another day though...)